### PR TITLE
[site][cicd] Add the Markdown style testing to 'rake test:cicd'

### PR DESCRIPTION
**Pull request by dshevtsov**
## Purpose of this pull request

This pull request:
- Adds a rake task 'test:style' that runs `mdl` on the entire repository.
- Adds the new task as a prerequisite to the `test:cicd` task.

### With no issues

1. Run the new task
```
$ rake test:style
```
Expect:
```
Testing Markdown style with mdl ...

No issues found
```

2. Run test:cicd

```
$ rake test:cicd
```

Expect:
```
Testing Markdown style with mdl ...

No issues found
Checking links with htmlproofer...
Running ["ScriptCheck", "ImageCheck", "LinkCheck", "LinkChecker::DoubleSlashCheck"] on ["_site"] on *.html... 

<truncated>
```

### With issues

Enable the first rule in `_checks/styles/style-rules-prod` by changing `exclude_rule 'MD001'` to `rule 'MD001'`

Run the `test:style` and `test:cicd` tasks again, but expect to see a report with issues in both cases.

## Additional information

The set of rules for `test:cicd` excludes all the rules, because each rule returns issues.
The rules should be enabled rule by rule like it was done for link checking project.

### CODE REVIEW

File: rakelib/check.rake [hunk 1]
```diff
@@ -24,6 +24,6 @@ namespace :check do
     puts 'Running Markdown linter ...'.magenta
     report = `bin/mdl #{path}`
     puts report.yellow
-    puts 'The rules are defined in _checks/md_style'.magenta
+    puts 'The rules are defined in _checks/styles/md_style'.magenta
```
Comment (jeff-matthews) [inline rakelib/check.rake]: should this be `_checks/styles/style-rules-dev`?

File: _checks/md_check_hook.rb [hunk 2]
```diff
@@ -1,6 +1,6 @@
 # This Jekyll hook runs Markdown linter (https://github.com/markdownlint/markdownlint)
 # on .md files that git tracks as 'modified' ('git ls-files -m').
-# '_checks/md_style' sets a style (https://github.com/markdownlint/markdownlint/blob/master/docs/creating_styles.md)
+# '_checks/styles/md_style' sets a style (https://github.com/markdownlint/markdownlint/blob/master/docs/creating_styles.md)
```
Comment (jeff-matthews) [inline _checks/md_check_hook.rb]: should this be `_checks/styles/style-rules-dev`?

File: rakelib/check.rake (hunk 1 above)
Comment (dshevtsov) [reply rakelib/check.rake]: fixed

File: _checks/md_check_hook.rb (hunk 2 above)
Comment (dshevtsov) [reply _checks/md_check_hook.rb]: fixed

**Comment by dshevtsov** [comment]
running tests

**Pull request merged**